File: Web/Scripts/Custom/models/Nodo.js
var Models;
(function (Models) {
    var Nodo = (function () {
        function Nodo(id, name) {
            if (id === void 0) { id = undefined; }
            if (name === void 0) { name = undefined; }
            this.Id = id;
            this.Name = name;
            this.Neighbors = [];
            this.Distances = [];
        }
        Nodo.prototype.addNeighbor = function (id, distance) {
            var temp = new Models.Neighbor(id, distance);
            this.Neighbors.push(temp);
        };
        return Nodo;
    })();
    Models.Nodo = Nodo;
})(Models || (Models = {}));

File: Web/Models/JS/Nodo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Web.Models.JS
{
    public class Nodo
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public List<Distance> Distances { get; set; }
        public List<Neighbor> Neighbors { get; set; }
        public Nodo()
        {
            Distances = new List<Distance>();
            Neighbors = new List<Neighbor>();
        }
    }
}